packages/fastify-podlet-server/README.md

```diff
@@ -570,3 +570,29 @@ export default {
 ```
 
 See the [Fastify docs](https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/) for more
+
+## Errors
+
+Error handler has been added to enable developers to control application errors.
+If you throw an error from app.setContentState or app.setFallbackState this will result in a 500 error. 
```

Comment:
```suggestion
If you throw an ordinary error from app.setContentState or app.setFallbackState this will result in the route serving a http 500 Internal Server error. 
```